### ISSUE TITLE
SSBC-EM3.2: SSBC goes Beta

### ISSUE THREAD

**Issue by eseliger** [sourcegraph/sourcegraph#26919]
**Outcome**: We feel comfortable not calling this a "tech demo" anymore. It will still have some features that are missing, some quirks here and there, but it is generally stable and good for use by anyone at this point, without a super big fat disclaimer that this is very early software and not intended for production use.

Wishes (not saying tasks until these become more concrete):
- [ ] We fix papercuts and open bugs
- [ ] We create user-facing docs for SSBC
- [ ] We have great, simple instructions on how to get an executor running
  - [ ] The setup burden is low
- [ ] The product behaves predictably and is fast
  - [ ] Ensure that SSBC and Batch Changes work well at scopes we haven't seen before. This unlocks 10k+ changeset campaigns
  - [ ] https://github.com/sourcegraph/sourcegraph/issues/26895
  - [ ] https://github.com/sourcegraph/sourcegraph/issues/26716
  - [ ] #26145
  - [ ] #26146

**Comment by malomarrec** [comment]
| Problem                                                                                                                                                       | Fix                                                                                                                                                   |
|---------------------------------------------------------------------------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------|
| I want to edit and re-execute a batch spec when I noticed it failed                                                                                           | Fixed by milestone 2.4                                                                                                                                |
| I want to see a list of all executions that doesn't require several clicks                                                                                    | Clarifying                                                                                                                                            |
| See all the batch changes that others ran                                                                                                                     | Fixed by milestone 2.4                                                                                                                                |
| I want to see all workspaces that completed, or errored, or have a certain status.                                                                            | Add basic filtering by status to the workspace execution list.                                                                                        |
| The Preview banner is not on the screen if I don't scroll.  If someone launches an execution and comes back, they might miss it and wonder what happens next. | Implement the layout from the [design](https://www.figma.com/file/cTKjwbvWqU6xAykZmCTTqy/SSBC?node-id=526%3A9554) with the Preview button at the top. |
| I want to know the position of my job in the queue, so I can act on it (ask for more executors for example).                                                  | Fixed in [3.2](https://github.com/sourcegraph/sourcegraph/issues/26919) but needs design @rrhyne                                                      |

**Comment by malomarrec** [comment]
@eseliger this sounds like a subset/implementation roadmap of the  Experimental success criteria documented here:
https://github.com/issues/assigned

**Comment by eseliger** [comment]
Yes, I think this makes sense, no? Jason's ticket is about the product milestone of getting to beta and this is a roadmap on how, at least that's how I read it :)

**Comment by malomarrec** [comment]
Woot woot! Bye bye em 3.4
